google-cloud-clients/google-cloud-firestore/src/test/java/com/google/cloud/firestore/TransactionTest.java

```diff
@@ -459,7 +459,7 @@ public QuerySnapshot updateCallback(Transaction transaction)
     assertEquals(3, requests.size());
 
     assertEquals(begin(), requests.get(0));
-    assertEquals(query(TRANSACTION_ID), requests.get(1));
+    assertEquals(query(TRANSACTION_ID, false), requests.get(1));
```

Comment:
I've forgotten what the second parameter is (and am too lazy to look it up.) Consider one of the options here:
http://go/java-practices/boolean-parameters

Reply:
Added `/* allDescendants= */` as a comment. Thanks